en/developer_guide/api.rst

```diff
@@ -0,0 +1,112 @@
+API
+===
+
+Api access
+----------
+
+To manage the connection to the API, Open Orchestra uses Oauth2 protocol.
```

Comment:
Slightly different : ``The connection to the API is based on the OAuth2 protocol``